tests/test_StatusReader/test_format_status.py

```diff
@@ -48,10 +48,17 @@
         pytest.param(application_4, f"Status: {GREEN_COMPLETED}"),
         pytest.param(orchestrator_1.shards[0], f"Status: {GREEN_RUNNING}"),
         pytest.param(orchestrator_1.shards[1], f"Status: {RED_FAILED}"),
-        pytest.param(pending_shard, f"Status: {StatusEnum.PENDING.value}"),
+        pytest.param(pending_shard, f"Status: {StatusEnum.UNKNOWN.value}"),
+        pytest.param(
+            malformed_status_dir_shard, f"Status: {StatusEnum.MALFORMED.value}"
+        ),
     ],
 )
 def test_get_status(entity: t.Dict[str, t.Any], expected_status):
-    status_dir = entity.telemetry_metadata["status_dir"]
-    status = format_status(get_status(status_dir))
+    try:
+        status_dir = entity.telemetry_metadata["status_dir"]
+        status = format_status(get_status(status_dir))
+    except KeyError:
+        assert expected_status == f"Status: {StatusEnum.MALFORMED.value}"
+        return
     assert status == expected_status
```

Comment:
Minor nit:
```suggestion
    try:
        status_dir = entity.telemetry_metadata["status_dir"]
    except KeyError:
        status = f"Status: {StatusEnum.MALFORMED.value}"
   else:
       status = format_status(get_status(status_dir))
    assert status == expected_status
```

Reply:
Also, where is this `KeyError` being rasied from? Is it from the `entity.telemetry_metadata["status_dir"]`? If so, it doesn't seem like this route is testing any of the application code.

Reply:
Per convo offline, it looks like the new parameters are not testing application code. Feel free to drop the new param and revert the `try/except` block

Reply:
Resolved at https://github.com/CrayLabs/SmartDashboard/pull/41/commits/b1842491092060c1bfac148b67e19f79d705f952